### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by Veekshith-Thoram**
image

### CODE REVIEW

**Review by DevRaahul — COMMENTED** [review]
1. Use color constant at all places where you used rgb() or hex value
2. In prev PR you have added same icons so you can move these icons to one common folder

File: lists/list.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+:root{
+    --primary-color: #34D399;
+}
+
+
+body{
+    margin: 0 auto;
+    font-family: 'Fredoka', sans-serif;
+}
```
Comment (kart-c) [inline lists/list.css]: The CSS styles written here most probably will apply to all your components. If this is getting repeated in other components as well, you can extract it into a different file and then import it wherever this code repeats.

File: lists/list.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    color: var(--primary-color);
+}
+.side-nav-link{
+    margin-top: 1rem;
+}
+
+
+.list-container{
+    display: flex;
+    flex-direction: row;
```
Comment (kart-c) [inline lists/list.css:6]: this is a default flex property. No need to specify. You can read about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#:~:text=Understanding%20WCAG%202.0-,Formal%20definition,-Initial%20value).

File: lists/list.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    <link rel="stylesheet" href="list.css">
+</head>
+
+<body>
+    <nav class="navigation">
+        <div class="nav-brand">Spud Ui</div>
+        <ul class="list-non-bullet nav-pills">
+            <li class="inline-item">
+                <a class="links" target="_blank" href="https://github.com/Veekshith-Thoram"><img
+                        src="/lists/images/icons8-github-30.png" id="git-img" alt="github"></a>
```
Comment (kart-c) [inline lists/list.html]: The id here is not being used anywhere. Is there any specific use for this id?

**Review by kart-c — APPROVED** [review]
You can attach a screenshot of the UI, instead of both the UI and the code. Also, a code sandbox link to look at UI would be good.

**Review by NagaSatyaGeeth — APPROVED** [review]

**Pull request merged**